lib/constants/stage-operators.js

```diff
@@ -90,25 +104,28 @@ const STAGE_OPERATORS = [
     name: '$documents',
     value: '$documents',
     label: '$documents',
-    dbOnly: true,
+    outputStage: false,
     score: 1,
-    env: [ ATLAS ],
+    env: [ ATLAS, ADL, ON_PREM ],
     meta: 'stage',
     version: '4.4.9',
     apiVersions: [1],
-    description: 'Returns literal results for a full-text search.',
+    namespaces: [ DATABASE ],
```

Comment:
Did you change the code in compass that uses dbOnly?

Reply:
I need this to be released first. I learned that we have also stages that would only work with certain types of collections, so probably this client code is easier: 

```
const namespaceType = isTimeseries : TIME_SERIES ? Boolean(sourceName) ? VIEW : COLLECTION;

const stageOperators = filter((op) => op.namespaces.include(namespaceType)) ...

```